k8s/k8s-aws-efs-csi-driver/_variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "md_metadata" {
+  type        = any
+  description = "md_metadata object"
+}
+
+variable "kubernetes_cluster" {
```

Comment:
Instead of passing in an artifact, let’s make this MD agnostic and just pass in the parts we need.

Reply:
This will be inconsistent with all the other modules we have

Reply:
+1 to Chris here. For the modules consumed in application-templates we're using `name` > `md_metadata` and making them simple. For the ones we use in our k8s clusters, we pass in `md_metadta` and other artifacts like `kubernetes_cluster`. Curious what the guidance is here.